### ISSUE TITLE
Modify the loss function to Focal Loss

### ISSUE THREAD

**Issue by PythonImageDeveloper** [experiencor/basic-yolo-keras#174]
Hi,
is it possible to modify this function to focal loss ? if so , please add this loss.

**Comment by experiencor** [comment]
@zeynali Yes, it's possible. The code of focal loss in pytorch is here https://github.com/marvis/pytorch-yolo2. Would love if you can contribute.

**Comment by zeynali** [comment]
I'm new in keras and DL , If possible you add this scripts to your code.

https://github.com/mkocabas/focal-loss-keras/blob/master/focal_loss.py
